src/javascript/app/ng-wig/ng-wig.component.spec.js

```diff
@@ -0,0 +1,263 @@
+describe('component: ngWig', () => {
+    let component;
+    let controller;
+    let scope;
+    let content;
+    let options;
+    let buttons;
+    let beforeExecCommand;
+    let afterExecCommand;
+    let element;
+    let attrs;
+    let pasteSpy;
+    let beforeExecSpy;
+    let afterExecSpy;
+    let mocks;
+    let window;
+    let compile;
+
+    beforeEach(module('ngWig'));
+
+    beforeEach(inject((_$componentController_, _$rootScope_, _$window_, _$compile_, _ngWigToolbar_) => {
+        window = _$window_;
```

Comment:
we are not on the same page. 
I'm telling you that you should rename "window", because it's not allowed to overwrite global window object (you can try it on console "window = 5")